stats/test/platform-routes.test.js

```diff
@@ -208,6 +208,23 @@ describe('Platform Routes HTTP request handler', () => {
     })
   })
 
+  beforeEach(async () => {
+    await pgPools.stats.query('DELETE FROM daily_reward_transfers')
+    await pgPools.stats.query('DELETE FROM daily_scheduled_rewards')
+    await pgPools.stats.query('DELETE FROM participants')
+
+    await pgPools.stats.query(`
+      INSERT INTO participants (id, participant_address)
+      VALUES
+        (1, 'to1'),
+        (2, 'to2'),
+        (3, 'to3'),
+        (4, 'address1'),
+        (5, 'address2'),
+        (6, 'address3')
+    `)
```

Comment:
Same here - use `mapParticipantsToIds`

Reply:
Do we actually need to prepare this mapping in advance? I would expect `givenDailyRewardTransferMetrics` to take care for that.